<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionItemWidget</class>
 <widget class="QWidget" name="SessionItemWidget">
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>72</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>72</height>
   </size>
  </property>
  <layout class="QHBoxLayout" name="rootLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QLabel" name="avatarLabel">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#CFCFCF; border-radius:8px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="textLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="headRow">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>会话名称</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="headSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>12:00</string>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#999999;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="summaryLabel">
       <property name="text">
        <string>最后一条消息内容预览</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#666666;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
